tdms/tests/unit/test_fields.cpp

```diff
@@ -88,12 +79,8 @@ TEST_CASE("Test that a split field can be allocated and zeroed"){
   }
 }
 
-TEST_CASE("Test setting a component of a vector field"){
```

Comment:
Why don't we have this guy?

Reply:
deleted the `construct_3d_array` function so removed the test case. I know – bad. will replace 😅